src/EA.Weee.Web/Areas/AATFReturn/Controllers/ReusedRemoveSiteController.cs

```diff
@@ -0,0 +1,114 @@
+﻿namespace EA.Weee.Web.Areas.AatfReturn.Controllers
+{
+    using System;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using System.Web.Mvc;
+    using EA.Prsd.Core.Mapper;
+    using EA.Weee.Api.Client;
+    using EA.Weee.Core.AatfReturn;
+    using EA.Weee.Requests.AatfReturn;
+    using EA.Weee.Requests.AatfReturn.Obligated;
+    using EA.Weee.Web.Areas.AatfReturn.Mappings.ToViewModel;
+    using EA.Weee.Web.Areas.AatfReturn.Requests;
+    using EA.Weee.Web.Areas.AatfReturn.ViewModels;
+    using EA.Weee.Web.Constant;
+    using EA.Weee.Web.Controllers.Base;
+    using EA.Weee.Web.Infrastructure;
+    using EA.Weee.Web.Services;
+    using EA.Weee.Web.Services.Caching;
+
+    public class ReusedRemoveSiteController : ExternalSiteController
+    {
+        private readonly Func<IWeeeClient> apiClient;
+        private readonly BreadcrumbService breadcrumb;
+        private readonly IWeeeCache cache;
+        private readonly IMap<ReturnAndAatfToReusedRemoveSiteViewModelMapTransfer, ReusedRemoveSiteViewModel> mapper;
+
+        public ReusedRemoveSiteController(Func<IWeeeClient> apiClient, BreadcrumbService breadcrumb, IWeeeCache cache, IMap<ReturnAndAatfToReusedRemoveSiteViewModelMapTransfer, ReusedRemoveSiteViewModel> mapper)
+        {
+            this.apiClient = apiClient;
+            this.breadcrumb = breadcrumb;
+            this.cache = cache;
+            this.mapper = mapper;
+        }
+
+        [HttpGet]
+        public virtual async Task<ActionResult> Index(Guid organisationId, Guid returnId, Guid aatfId, Guid siteId)
+        {
+            using (var client = apiClient())
+            {
+                var sites = await client.SendAsync(User.GetAccessToken(), new GetAatfSite(aatfId, returnId));
+                var site = sites.AddressData.Select(s => s).Where(s => s.Id == siteId).Single();
+
+                var viewModel = mapper.Map(new ReturnAndAatfToReusedRemoveSiteViewModelMapTransfer()
```

Comment:
Add unit tests,
1. Check that the mapper is called with the correct values
2. Viewmodel is returned from the action with the correct values set